app/models/agents/data_output_agent.rb

```diff
@@ -160,7 +160,7 @@ def push_hubs
       interpolated['push_hubs'].presence || []
     end
 
-    def receive_web_request(params, method, format)
+    def receive_web_request(params, method, format, headers={}, request=ActionDispatch::Request.new({}))
```

Comment:
This could just be `receive_web_request(params, method, format)`, right?  Since we support both?